packages/react-components/react-nav-preview/src/components/Nav/useNav.ts

```diff
@@ -83,5 +92,62 @@ export const useNav_unstable = (props: NavProps, ref: React.Ref<HTMLDivElement>)
     onUnregister,
     onSelect,
     getRegisteredNavItems,
+    onRequestNavCategoryItemToggle,
+    openItems,
   };
 };
+
+// /**
+//  * Initial value for the uncontrolled case of the list of open indexes
+//  */
+// function initializeUncontrolledOpenItems({ defaultOpenItems }: Pick<NavProps, 'defaultOpenItems'>): NavItemValue[] {
+//   if (defaultOpenItems !== undefined) {
+//     if (Array.isArray(defaultOpenItems)) {
+//       return [defaultOpenItems[0]];
+//     }
+//     return [defaultOpenItems];
+//   }
+//   return [];
+// }
+
+// /**
+//  * Normalizes Accordion index into an array of indexes
+//  */
+// function normalizeValues(index?: NavItemValue | NavItemValue[]): NavItemValue[] | undefined {
+//   if (index === undefined) {
+//     return undefined;
+//   }
+//   return Array.isArray(index) ? index : [index];
+// }
+
+// temp implementation of the above function.
+function normalizeValues(index?: NavItemValue | NavItemValue[]): NavItemValue[] | undefined {
```

Comment:
It is better to define functions as const normalizeValues = (...) and avoid any unlikely binding issues

Reply:
🫡